--- a/CMakeLists.txt	2020-06-27 21:39:29.000000000 +0200
+++ b/CMakeLists.txt	2020-11-01 22:07:33.724321071 +0100
@@ -255,19 +255,6 @@
 	set_target_properties(TheDarkMod PROPERTIES WIN32_EXECUTABLE TRUE)
 elseif (UNIX)
 	target_link_libraries(TheDarkMod pthread dl stdc++fs X11 Xext Xxf86vm)
-	# strip debug symbols from executable and put them into a separate symbols file
-	add_custom_command(
-			TARGET TheDarkMod POST_BUILD
-			COMMAND objcopy --only-keep-debug "$<TARGET_FILE:TheDarkMod>" "$<TARGET_FILE:TheDarkMod>.debug"
-	)
-	add_custom_command(
-			TARGET TheDarkMod POST_BUILD
-			COMMAND strip --strip-debug --strip-unneeded "$<TARGET_FILE:TheDarkMod>"
-	)
-	add_custom_command(
-			TARGET TheDarkMod POST_BUILD
-			COMMAND objcopy "--add-gnu-debuglink=$<TARGET_FILE:TheDarkMod>.debug" "$<TARGET_FILE:TheDarkMod>"
-	)
 endif ()
 
 if (COPY_EXE)
