diff -Naur a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt	2021-02-13 18:38:55.890729485 +0100
+++ b/CMakeLists.txt	2021-02-13 18:38:56.374728593 +0100
@@ -61,14 +61,15 @@
 endif ()
 
 # In Ubuntu 13.04, the built-in check for python fails.. so we have to resort to the check with pkg-config (which doesn't work in many distros)
-find_package(PythonLibs 2.6)
-if (NOT PYTHONLIBS_FOUND)
-    pkg_check_modules(PYTHON REQUIRED python)
+find_package(Python2 COMPONENTS Development)
+if (NOT Python2_FOUND)
 endif ()
 
 
 # We need to be able to detect python module versions and packages hence this function
-find_program(PYTHON_EXEC "python2")
+set(PYTHON_INCLUDE_DIRS ${Python2_INCLUDE_DIRS})
+set(PYTHON_LIBRARIES ${Python2_LIBRARIES})
+set(PYTHON_EXEC ${Python2_EXECUTABLE})
 include(FindPackageHandleStandardArgs)
 
 find_package(MySQL 5.6.0 REQUIRED)
diff -Naur a/library/grt/src/CMakeLists.txt b/library/grt/src/CMakeLists.txt
--- a/library/grt/src/CMakeLists.txt	2021-02-13 18:38:56.129729045 +0100
+++ b/library/grt/src/CMakeLists.txt	2021-02-13 18:38:56.337728661 +0100
@@ -49,7 +49,7 @@
     ${UUID_LIBRARIES}
   PRIVATE 
     ${MySQLCppConn_LIBRARIES}
-    ${PYTHON_LIBRARIES}
+    ${Python2_LIBRARIES}
 )
 
 add_library(grt_Iface INTERFACE)
