--- a/configuration.example.ini 00:10:32.747109613 +0100
+++ b/configuration.example.ini 00:27:27.623741195 +0100
@@ -35,8 +35,8 @@
 #    This option will be overridden by command line arguments.
 # 'music_database_path': The path of database that stores the music library. Can be disabled by
 #    setting 'save_music_library=False'
-#database_path=settings.db
-#music_database_path=music.db
+database_path=/var/lib/botamusique/settings.db
+music_database_path=/var/lib/botamusique/music.db
 
 # 'admin': Users allowed to kill the bot, or ban URLs. Separated by ';'
 #admin = User1;User2;
